.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "emData/project_generation_scripts"]
 	path = emData/project_generation_scripts
-	url = https://github.com/cms-L1TK/project_generation_scripts.git
+	url = https://github.com/cms-L1TK/project_generation_scripts
```

Comment:
@aryd I guess the convention is to include the `.git` extension, although it isn't required, so I think we should revert this change:
```suggestion
	url = https://github.com/cms-L1TK/project_generation_scripts.git
```

Reply:
Fixed